backend/main.py

```
@@ -110,4 +108,111 @@ async def analyze_glaucoma(file: UploadFile = File(...)):
+                image = image.convert('RGB')
+                
+            current_dir = os.path.dirname(os.path.abspath(__file__))
+            root_dir = os.path.dirname(current_dir)
+            ai_dir = os.path.join(root_dir, 'ai')
+            
+            if ai_dir not in sys.path:
+                sys.path.append(ai_dir)
+                
+            from pipeline.pipeline import GlaucomaPipeline
+            
+            global glaucoma_pipeline
```

Ruff (0.15.9)

[warning] Using the global statement to update `glaucoma_pipeline` is discouraged

(PLW0603)